### PR TITLE
Add Jaakko-Os smoshiur1237 and fgofurov7 as reviewers

### PR DESCRIPTION
**Pull request by furkatgofurov7**

### CODE REVIEW

**Review by Jaakko-Os — COMMENTED** [review]
Excellent

File: OWNERS [hunk 1]
```diff
@@ -12,3 +12,6 @@ reviewers:
  - kashifest
  - jan-est
  - xenwar
+ - Jaakko-Os
```
Comment (maelk) [inline OWNERS:15]: Not sure if the capitalization matters. but all other are lower case.

**Review by maelk — APPROVED** [review]
Notifying the other owners on this @dhellmann @russellb @hardys @stbenjam and @fmuyassarov

File: OWNERS (hunk 1 above)
Comment (furkatgofurov7) [reply OWNERS:15]: It doesn't I guess, the capitalized name is already in other [repository](https://github.com/metal3-io/cluster-api-provider-metal3/blob/f9f2b3394d02942821819ffcd9a3e7324a5ab037/OWNERS#L13) owners file.

**Review by smoshiur1237 — COMMENTED** [review]
Thanks

**Review by fmuyassarov — COMMENTED** [review]
Thanks for joining!

**Comment by russellb** [comment]
lgtm

**Pull request merged**